Packs/GenericSQL/Integrations/GenericSQL/GenericSQL.py

```diff
@@ -152,13 +152,14 @@ def sql_query_execute_request(self, sql_query: str, bind_vars: Any, fetch_limit=
             sql_query = text(sql_query)
 
         result = self.connection.execute(sql_query, bind_vars)
-        # For avoiding responses with lots of records
-        results = result.fetchmany(fetch_limit) if fetch_limit else result.fetchall()
+        results_as_dict = result.mappings().all()
+        converted_results = [dict(row) for row in results_as_dict]
```

Comment:
```suggestion
        results = [dict(row) for row in result.mappings().all()]
```
change `converted_results` to `results` elsewhere, `converted_` is mostly redundant